Makefile

```diff
@@ -24,7 +25,12 @@ run:
 	cd ./game && go run .
 
 
-build: generate build-windows build-linux
+BUILD_NATIVE = build-linux
+ifeq ($(shell uname -s), Darwin)
+	BUILD_NATIVE = build-macos
+endif
+
+build: generate build-windows ${BUILD_NATIVE}
```

Comment:
The build command should build for all supported platforms. So the correct change here would be to simply add build-darwin as well

Reply:
We should also keep build-linux

Reply:
I've made the change (and deleted the platform detection part). I don't know whether Darwin builds can be done under Linux without `CC` set. If so, this indeed should be preferred.